app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -177,25 +161,6 @@
 .gem-c-action-link--light-text {
   color: govuk-colour("white");
 
-  .gem-c-action-link__link,
-  .gem-c-action-link__subtext-link {
-    text-decoration: underline;
-
-    &:link,
-    &:visited {
-      color: govuk-colour("white");
-    }
-
-    &:hover {
-      color: $gem-hover-dark-background;
-    }
-
-    &:focus {
-      text-decoration: none;
-      color: $govuk-focus-text-colour;
-    }
```

Comment:
This might be still required, as I spotted an invisible link on the _[With link on subtext on a dark background](https://govuk-publis-component--waebdu.herokuapp.com/component-guide/action_link/with_link_on_subtext_on_a_dark_background/preview)_ variation:

![image](https://user-images.githubusercontent.com/1732331/118805581-fb465980-b89d-11eb-9877-9bdd511177ba.png)
![image](https://user-images.githubusercontent.com/1732331/118805633-0c8f6600-b89e-11eb-8aef-a50237e8b15d.png)

Reply:
Good spot 🤣  Will sort out